sound/soc/sof/intel/hda-dai.c

```diff
@@ -318,25 +318,9 @@ static int hda_dai_trigger(struct snd_pcm_substream *substream, int cmd, struct
 	return 0;
 }
 
-static int hda_dai_hw_free(struct snd_pcm_substream *substream,
-			   struct snd_soc_dai *dai)
-{
-	struct snd_soc_dapm_widget *w = snd_soc_dai_get_widget(dai, substream->stream);
-	struct snd_sof_dai_config_data data = { 0 };
-	int ret;
-
-	ret = hda_link_dma_hw_free(substream, dai);
-	if (ret < 0)
-		return ret;
-
-	data.dai_data = DMA_CHAN_INVALID;
-
-	return hda_dai_config(w, SOF_DAI_CONFIG_FLAGS_HW_FREE, &data);
-}
-
 static const struct snd_soc_dai_ops hda_dai_ops = {
 	.hw_params = hda_dai_hw_params,
-	.hw_free = hda_dai_hw_free,
+	.hw_free = hda_link_dma_hw_free,
```

Comment:
Can we rename the `hda_link_dma_hw_free` to `hda_dai_hw_free` as well?
It looks completely out of place in this struct with the other callbacks are sans `_link_`.

Reply:
done. thanks!